docs/diagrams/data-flow/README.md

```diff
@@ -6,12 +6,13 @@ The purpose of this diagram is to show how data flows through the Privacy Protec
 - The yellow ovals are kafka topics that the PPM consumes from and produces to.
 
 ## Data Flow Explanation
-1. The PPM pulls from the Asn1 Input Topic and passes the messge to the BSMHandler class.
+1. The PPM pulls from the Input Topic and passes the message to the BSMHandler class.
 1. Throughout the flow, if the BSMHandler encounters a parse error or missing data, the message will be dropped.
```

Comment:
Can we copy the flow chart to the readme.md so we can see the flow chart and explanation at one page?

Reply:
Yup! Once https://github.com/CDOT-CV/jpo-cvdp/pull/22 is merged this will have been addressed.

Reply:
The data flow chart has been added to the data flow README.